crates/bench-api/src/lib.rs

```diff
@@ -238,19 +239,25 @@ impl WasmBenchConfig {
         Ok(Some(stdin_path.into()))
     }
 
-    fn execution_flags(&self) -> Result<Option<CommonOptions>> {
-        if self.execution_flags_ptr.is_null() {
-            return Ok(None);
-        }
-
-        let execution_flags = unsafe {
-            std::slice::from_raw_parts(self.execution_flags_ptr, self.execution_flags_len)
+    fn execution_flags(&self) -> Result<CommonOptions> {
+        let flags = if self.execution_flags_ptr.is_null() {
+            "--disable-cache".to_string()
+        } else {
+            let execution_flags = unsafe {
+                std::slice::from_raw_parts(self.execution_flags_ptr, self.execution_flags_len)
+            };
+            let execution_flags = std::str::from_utf8(execution_flags)
+                .context("given execution flags string is not valid UTF-8")?;
+            format!("--disable-cache {execution_flags}")
```

Comment:
Can we just do a null pointer and zero len for flags when not provided, always parse the `Config` from that resulting potentially-empty flags string, and then rely on https://github.com/bytecodealliance/wasmtime/pull/5542 for disabling the cache?